### ISSUE TITLE
Scatter plot for training data

### ISSUE THREAD

**Issue by egorvlgavr** [egorvlgavr/ltr-utils#1]
It may be good to have ability to view how two/three attributes are correlated with each other.
Create option in **analyse_data.py** This option should be enabled only for training set. Draw plot with **matlabplotlib**. As axis values should be supported **rank** and feature values. If defined both X, Y, Z axis draw 3D plot. If defined only X,Y draw 2D plot.

**Comment by egorvlgavr** [comment]
Update read me based on new functionality
